src/Backend/Core/Engine/DataGrid.php

```diff
@@ -446,7 +446,7 @@ public function setColumnConfirm($column, $message, $custom = null, $title = nul
      * @param string[]|string $columns The column wherein the result will be printed.
      * @param bool $overwrite Should the original value be overwritten.
      */
-    public function setColumnFunction($function, $arguments, $columns, $overwrite = true): void
+    public function setColumnFunction($function, $arguments = null, $columns = null, $overwrite = true): void
```

Comment:
Beetje stom dat daar ook een task van deploy.php in zit. Ken je `git add -p`? Dan kan je beslissen per chunk of die moet gecommit worden?